### PR TITLE
Resolves #96

### LINKED ISSUE #96: Create an error when trying to create a room name that is already in use

### PR DESCRIPTION
**Pull request by jni77**

### CODE REVIEW

File: src/components/StreamerConnection.vue [hunk 1]
```diff
@@ -143,6 +143,11 @@ export default {
         } catch (e) {}
       };
 
+      this.connection.onUserIdAlreadyTaken = (useridAlreadyTaken) => {
+        this.$emit('idTaken', useridAlreadyTaken);
+        // alert(useridAlreadyTaken + ' already taken. Please choose another room name.');
```
Comment (JohnmarkBeaty) [inline src/components/StreamerConnection.vue]: Let's delete this commented out code!

File: src/views/Streamer.vue [hunk 2]
```diff
@@ -154,11 +154,13 @@
     :roomPassword="room_password"
     @sessionId="onSessionId"
     @viewerCount="onViewerCount"
+    @idTaken="onIdTaken"
   )
   .col-md-1-2
     LogoSvg#logo
     #live-indicator(:class="{ live: isSharingOn && sessionId }") LIVE
   .col-md-1-2
+    div#id-taken(v-if="useridAlreadyTaken") {{useridAlreadyTaken}} already taken!
```
Comment (JohnmarkBeaty) [inline src/views/Streamer.vue]: Can we make this text red so it's more noticeable? 
Also, let's use the text from above `Whoops, useridAlreadyTaken + ' already taken. Please choose another room name.'`

File: src/components/StreamerConnection.vue [hunk 3]
```diff
@@ -143,6 +143,11 @@ export default {
         } catch (e) {}
       };
 
+      this.connection.onUserIdAlreadyTaken = (useridAlreadyTaken) => {
```
Comment (JohnmarkBeaty) [inline src/components/StreamerConnection.vue]: So I am noticing that when I test this it _does_ start a stream. 
I'm not sure if we can check before the share screen pops up? If not, we should immediately end the stream.

File: src/components/StreamerConnection.vue (hunk 1 above)
Comment (jni77) [reply src/components/StreamerConnection.vue]: Oops, fixed!

File: src/views/Streamer.vue (hunk 2 above)
Comment (jni77) [reply src/views/Streamer.vue]: I've gone and added some styling to the alert.

File: src/components/StreamerConnection.vue (hunk 3 above)
Comment (jni77) [reply src/components/StreamerConnection.vue]: This is probably outside the scope of my abilities and should probably be a new issue. It doesn't start a new room, so the tab is streaming into the unknown.

File: src/views/Streamer.vue [hunk 4]
```diff
@@ -154,11 +162,14 @@
...
     @sessionId="onSessionId"
     @viewerCount="onViewerCount"
+    @idTaken="onIdTaken"
   )
   .col-md-1-2
     LogoSvg#logo
     #live-indicator(:class="{ live: isSharingOn && sessionId }") LIVE
   .col-md-1-2
+    div#id-taken(v-if="useridAlreadyTaken")
+      |⚠️ Whoops, {{useridAlreadyTaken}} already taken! Please choose another room name.
```
Comment (JohnmarkBeaty) [inline src/views/Streamer.vue]: Lets Bold the room name (`{{useridAlreadyTaken}}`) so it stands out a bit!

**Pull request merged**